src/pytorch_lightning/loggers/csv_logs.py

```diff
@@ -161,15 +161,15 @@ def log_dir(self) -> str:
         return log_dir
 
     @property
-    def save_dir(self) -> Optional[str]:
+    def save_dir(self) -> str:
         """The current directory where logs are saved.
 
         Returns:
             The path to current directory where logs are saved.
         """
         return self._save_dir
 
-    @property
+    @property  # type: ignore[misc]
```

Comment:
why do you need to ignore the type here?

Reply:
mypy complains about "decorated property types not supported"